pkg/health/health.go

```diff
@@ -82,6 +91,40 @@ func Run(bird, bird6, felixReady, felixLive bool, thresholdTime time.Duration) {
 	}
 }
 
+func checkBIRDLive() error {
+	error := checkService("confd")
+	if error != nil {
+		return error
+	}
+
+	error = checkService("bird")
```

Comment:
I don't think we always run both bird and bird6.  Certainly the readiness implementation (with separate flags) suggests that.

Reply:
I will add extra flags for bird6.